### PR TITLE
Disable NavigationThreadingOptimizations on Desktop for 5% of users - Production

### PR DESCRIPTION
**Pull request by bsclifton**
## Test plan
To reproduce (on a bad profile). Run the following WITH and WITHOUT the study:
1. open [news.ycombinator.com](http://news.ycombinator.com/)
2. open devtools
3. go to network tab
4. check "disable cache"
5. Refresh
6. Observe gap between first request and subsequent requests

Here is a bad profile you can use 😄 
[slow-profile.zip](https://github.com/brave/brave-variations/files/8051775/slow-profile.zip)

## Description
Uplift of staging version merged with https://github.com/brave/brave-variations/pull/216 / https://github.com/brave/brave-variations/pull/218 / https://github.com/brave/brave-variations/pull/219

NOTE: do not merge yet. We need to finish testing and figure out what percentages to roll out

> This is currently disabled on Chrome, at least for version 98 and for desktop. I am unsure if this problem affects Android or when we update to Chromium 99, so I've left it as default for android and desktop with cr99+ (enabled in brave-core/chromium).
> 
> Fix: https://github.com/brave/brave-browser/issues/21029

### CODE REVIEW

File: seed/seed.json
Comment (iefremov) [inline seed/seed.json:1061]: just drop this

Comment (bsclifton) [reply seed/seed.json]: Done! 😄

**Comment by kjozwiak** [comment]
Verification **PASSED** using the STR/Cases outlined via https://github.com/brave/brave-variations/pull/217#issue-1131748258.

* using the profile from above, launched the profile using `Brave Nightly` and noticed the performance issue ~200ms
* re-launched the same profile using `brave.exe --enable-logging=stderr --v=1 --variations-server-url=https://variations.bravesoftware.com/seed` and ensured that the performance issue wasn't occurring anymore

**`Example (Performance Issue)`** | **`Example (Resolved using "NavigationThreadingOptimizationsCompat:Disabled")`**
--------------------------------------|-----------------------------------------------------------------------------------------------
image | image (1)

**Review by petemill — APPROVED** [review]
lgtm

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers. QA has verified the PR as per https://github.com/brave/brave-variations/pull/217#issuecomment-1036657679.

**Pull request merged**
